skrifa/src/outline/glyf/hint/engine/graphics_state.rs

```diff
@@ -0,0 +1,1130 @@
+//! Managing the graphics state.
+//!
+//! See <https://learn.microsoft.com/en-us/typography/opentype/spec/tt_instructions#managing-the-graphics-state>
+
+// 45 instructions
```

Comment:
maybe this can be part of the mod docs?

Reply:
Makes sense. I'll update the other modules as well.